packages/fxa-content-server/tests/functional/sign_in_blocked.js

```diff
@@ -171,12 +171,13 @@ registerSuite('signin blocked', {
           )
 
           .then(fillOutSignInUnblock(email, 0))
-          // the first code is no longer valid, must use the 2nd.
-          .then(visibleByQSA(selectors.SIGNIN_UNBLOCK.ERROR))
-          .then(testErrorTextInclude('invalid'))
 
+          // If these checks are in place, the edge case where we have swapped primary accounts fails.
```

Comment:
Not sure about this. As we discussed it sounds like burning the unblock code even on a bad login attempt is intentional. So this probably isn't acceptable and another approach should be taken.